packages/templates/wasm/assemblyscript/meta/test.graphql

```diff
@@ -1,5 +1,5 @@
-mutation {
-  deployContract(
-    connection: $connection
+query {
+  sampleMethod(
+    arg: $arg
   )
```

Comment:
Why do we still have meta manifest queries? @krisbitney @cbrzn @dOrgJelli?

Reply:
I think we should replace them with workflows but as far as I know that just hasn't been done yet. Should I just remove them from the template for now?